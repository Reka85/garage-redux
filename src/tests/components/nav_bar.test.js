import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import NavBar from "../../components/nav_bar";

test("should render NavBar correctly", () => {
  const wrapper = shallow(<NavBar garage={'MyGarage'}/>);
  expect(wrapper).toMatchSnapshot();
});
