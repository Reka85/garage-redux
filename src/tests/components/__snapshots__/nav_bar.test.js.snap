// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NavBar correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar
    garage="MyGarage"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        replace={false}
        to="/"
      >
        <i
          className="fas fa-wrench"
        />
         
        MyGarage
      </Link>,
      "className": "bar",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <i
            className="fas fa-wrench"
          />,
          " ",
          "MyGarage",
        ],
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "fas fa-wrench",
          },
          "ref": null,
          "rendered": null,
          "type": "i",
        },
        " ",
        "MyGarage",
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          replace={false}
          to="/"
        >
          <i
            className="fas fa-wrench"
          />
           
          MyGarage
        </Link>,
        "className": "bar",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <i
              className="fas fa-wrench"
            />,
            " ",
            "MyGarage",
          ],
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fas fa-wrench",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          " ",
          "MyGarage",
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
